tests/unit/helpers/ago-test.js

```diff
@@ -1,17 +1,19 @@
-import date from './date';
 import ago from 'ember-moment/helpers/ago';
+import date from './date';
 import callHelper from '../../helpers/call-helper';
 
 module('AgoHelper');
 
-var FAKE_HANDLEBARS_CONTEXT = {};
+let FAKE_HANDLEBARS_CONTEXT = {};
+let threeDaysAgo = new Date();
+threeDaysAgo.setDate(threeDaysAgo.getDate() - 3);
 
-test('one arg (date)', function() {
-  equal(callHelper(ago, [date(0), FAKE_HANDLEBARS_CONTEXT]), "45 years ago");
+test('one arg (date)', () => {
```

Comment:
Test was fragile, could be 45 years or 46 years depending on environment test is run in